src/luacheck/builtin_standards/ngx.lua

```diff
@@ -146,6 +146,8 @@ local ngx_defs = {
             set_var = {other_fields = true},
          },
       },
+      table = standards.def_fields("clear", "clone", "isarray", "isempty", "nkeys"),
```

Comment:
does this list add to or replace the `"table"` rules? I see that the `"string"` entry above includes all standard `string.*`  APIs

Reply:
@hishamhm 
Look like the list is added but not overwritten.
The `string` entry above is attached to `ngx.config.subsystem`, not applied to the global variable.